wp-multi-network/includes/classes/class-wp-ms-networks-api.php

```diff
@@ -0,0 +1,859 @@
+<?php
+/**
+ * REST API: WP_REST_Networks_Controller class
+ *
+ * @package WPMN
+ * @subpackage REST_API
+ * @since 2.2.0
+ */
+
+/**
+ * Core controller used to access networks via the REST API.
+ *
+ * @since 2.2.0
+ *
+ * @see WP_REST_Controller
+ */
+class WP_REST_Networks_Controller extends WP_REST_Controller {
+
+	/**
+	 * Constructor.
+	 *
+	 * @since 2.2.0
+	 */
+	public function __construct() {
+		$this->namespace = 'wpmn/v1';
+		$this->rest_base = 'networks';
+	}
+
+	/**
+	 * Registers the routes for the objects of the controller.
+	 *
+	 * @since 2.2.0
+	 */
+	public function register_routes() {
+
+		register_rest_route(
+			$this->namespace, '/' . $this->rest_base, array(
+				array(
+					'methods'   => WP_REST_Server::READABLE,
+					'callback'  => array( $this, 'get_items' ),
+					'permission_callback' => array( $this, 'get_items_permissions_check' ),
+					'args'      => $this->get_collection_params(),
```

Comment:
Please make sure arrows are aligned correctly in the arrays (same thing for multiple arrays in this method).